driver/src/contracts/stablex_contract_auction_element.rs

```diff
@@ -0,0 +1,224 @@
+use byteorder::{BigEndian, ByteOrder};
+
+use std::collections::HashMap;
+
+use web3::types::{H160, U256};
+
+use dfusion_core::models::{BatchInformation, Order};
+
+#[derive(Debug, Default, PartialEq)]
+pub struct AuctionElement {
```

Comment:
I think this is the right place for this struct, but I'd try to name it in accordance with the file. Maybe it should be a `StableXAuctionElement`? And the file could be called `stablex_auction_element`